data/ABRS_2023/metadata.yml

```diff
@@ -349,6 +352,36 @@ traits:
   basis_of_value: measurement
   replicates: .na
   methods: Maximum plant vegetative height of a mature individual.
+- var_in: plant_photosynthetic_organ
+  unit_in: mm
+  trait_name: plant_photosynthetic_organ
+  entity_type: species
+  value_type: value_type
+  basis_of_value: measurement
```

Comment:
Should these be `expert_score` as they are categorical values (relevant to the rest of the studies)?

Reply:
yes, I didn't copy and paste correctly